.github/workflows/check-deep-tests-reusable.yml

```diff
@@ -19,4 +21,7 @@ jobs:
       with:
         script: |
           const script = require('${{ github.workspace }}/dafny/.github/workflows/check-for-workflow-run.js')
-          console.log(script({github, context, core}, workflowID = 'deep-tests.yml', branch = 'master', sha = '${{ inputs.sha }}'))
+          console.log(script({github, context, core},
+                             {workflow_id: 'deep-tests.yml',
+                              sha: '${{ inputs.sha }}' || null,
```

Comment:
The `|| null` seems redundant to me, since `''` is equally falsey.

Reply:
We don't know whether the library treats them the same, do we?

Reply:
Solved by removing the keys from the object completely when they are not given.